Autopilot/AttitudeManager/PWM.h

```diff
@@ -31,18 +31,32 @@
 #define MIN_PWM -1024
 
 /**
+ * Used in some of the calculations
+ */
+#define HALF_PWM_RANGE (MAX_PWM - MIN_PWM)/2
+
+/**
  * Maximum and minimum limits received from the controller. These should be calibrated,
  * and the scaling from these values to the MAX and MIN PWM values will depend on these.
  * Note that these will be used as the default/initial values, however the actual scaling values
  * can be set by the ground station
  */
-#define UPPER_PWM 1284
-#define LOWER_PWM 642
+#define UPPER_PWM 1250
+#define LOWER_PWM 625
 
 /**
- * Used in some of the calculations
+ * The middle of the PWM range of the RC controller. This is used as the initial
+ * offset in the output and input scaling calculations
  */
-#define HALF_PWM_RANGE (MAX_PWM - MIN_PWM)/2
+#define MIDDLE_PWM (int)(((UPPER_PWM - LOWER_PWM)/2) + LOWER_PWM)
```

Comment:
Same reasoning as to why you would declare variables inside files static, you'd do the same with defines. It tells the programmer that the scope of that define only exists within this file, so he can rename it, modify it, etc without worrying adverse effects.